src/routes/mods.rs

```diff
@@ -78,6 +86,14 @@ pub async fn mod_get(
             description: m.description,
             body_url: m.body_url,
             published: m.published,
+            updated: m.updated,
+            status: match m.status.0 {
```

Comment:
Same as the above.  Also, this is a significant amount of duplication; we should move the `database::models::Mod` to `models::Mod` conversion to a separate function that we can reuse.